src/handler/target.rs

```diff
@@ -599,24 +603,6 @@ pub struct TargetConfig {
     pub cache_enabled: bool,
 }
 
-impl TargetConfig {
-    pub fn new(
-        ignore_https_errors: bool,
-        request_timeout: Duration,
-        viewport: Option<Viewport>,
-        request_intercept: bool,
-        cache_enabled: bool,
-    ) -> Self {
-        Self {
-            ignore_https_errors,
-            request_timeout,
-            viewport,
-            request_intercept,
-            cache_enabled,
-        }
-    }
-}
```

Comment:
this is unrelated?

Reply:
It is. My bad. I just suddenly started refactoring your crate at one point.